.github/resources/codeql-config.yml

```diff
@@ -0,0 +1,6 @@
+name: "CodeQL configuration"
+
+paths-ignore:
```

Comment:
paths-ignore does not work for C++ and Java sources as yet - https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#specifying-directories-to-scan